_pages/cheri-feature-matrix.md

```diff
@@ -133,17 +133,17 @@ flow:
             | Kernel temporal safety | - | - |
             | KGDB support for memory-safe kernels | Yes | -[^1] |
             | KGDB support for memory-unsafe kernels | Yes | - (as above) |
-            | Userspace spatial safety<br />(w/memory-unsafe kernel) | Yes | Yes
+            | Userspace spatial safety<br />(w/memory-unsafe kernel) | Yes | Yes |
             | Userspace spatial safety<br />(w/memory-safe kernel) | Yes | - |
-            | Userspace temporal safety | Experimental<br />(branch - expected 23.06 release) | - |
+            | Userspace temporal safety | Experimental<br />([caprevoke](https://github.com/CTSRD-CHERI/cheripedia/wiki/HOWTO:-Use-Cornucopia-with-the-22.12-CheriBSD-Release) - expected 23.06 release) | - |
```

Comment:
I wonder if we should link both the branch and the instructions, maybe something like
```suggestion
            | Userspace temporal safety | Experimental<br />([caprevoke](https://github.com/CTSRD-CHERI/cheribsd/tree/caprevoke)- [partial support](https://github.com/CTSRD-CHERI/cheripedia/wiki/HOWTO:-Use-Cornucopia-with-the-22.12-CheriBSD-Release) in 22.12 - expected 23.06 release) | - |
```

Reply:
That would be ideal but I'm afraid we don't have enough space for that. The CheriBSD column is already pretty wide and we'd have to break the line here. I know that the branch name in place of the tutorial link isn't perfect but I think it looks the best.

Before: https://drive.google.com/file/d/1-2FPVyRATzU6clm7l94XwwoQj9rv6-Rh/view?usp=sharing
After: https://drive.google.com/file/d/1IJU5RNy3plWAfnksRxn4GWdapXsFCLrz/view?usp=sharing
After (alternative version): https://drive.google.com/file/d/1DSUbzzDJF0UigbH8nWeqt0YDX4Gw3fpF/view?usp=sharing

Reply:
I moved information on the next release to footnotes so that there's less text in the table.